src/mame/igs/goldstar.cpp

```diff
@@ -7185,51 +7181,47 @@ static INPUT_PORTS_START( fb2010 )  // hit 'start1' to init NVRAM for first time
 
 	// definitions and defaults taken from manual (unknown if manual is for this exact version)
 	PORT_START("DSW1")
-	PORT_DIPNAME( 0x07, 0x00, DEF_STR( Difficulty ) ) PORT_DIPLOCATION("DSW1:1,2,3")  // 'Game Level' in manual
-	PORT_DIPSETTING(    0x00, "Level 8 (Hardest)" )
-	PORT_DIPSETTING(    0x01, "Level 7" )
-	PORT_DIPSETTING(    0x02, "Level 6" )
-	PORT_DIPSETTING(    0x03, "Level 5" )
-	PORT_DIPSETTING(    0x04, "Level 4" )
-	PORT_DIPSETTING(    0x05, "Level 3" )
-	PORT_DIPSETTING(    0x06, "Level 2" )
+	PORT_START("DSW1")
```

Comment:
you have two PORT_START("DSW1") after another (line 7183 and 7184). This breaks validation.